addons/mail/static/src/views/fields/properties_field/property_value.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module **/
+
+import { PropertyValue } from "@web/views/fields/properties/property_value";
+import { useOpenChat } from "@mail/web/open_chat_hook";
+import { patch } from "@web/core/utils/patch";
+import { TagsList } from "@web/views/fields/many2many_tags/tags_list";
+
+// Many2one
+patch(PropertyValue.prototype, "mail.PropertyValue", {
+    setup() {
+        this._super();
+        if (this.env.services.messaging) {
+            // work only for the res.users model
+            this.openChat = useOpenChat("res.users");
+        }
+    },
+
+    onAvatarClick() {
```

Comment:
Not important: It seems like they used the name `_onAvatarClicked` for that callback function, I would maybe use the same name as them ? (there is not tones of results when grepping that function name).

Reply:
yes ok :)